### PR TITLE
Initial checkin of csv_to_kdi.py

### PR DESCRIPTION
**Pull request by rick-kenna**
Still testing, but it is pretty complete.  It is based of of `csv_KDI_json.rb` with some enhancements in argument processing.

If you think logging would be nice, please speak up.

I found a "main locator" bug.

### CODE REVIEW

**Review by jaredkalmus — CHANGES_REQUESTED** [review]
Output is not matching between the Ruby script and Python script. My metafile has tag columns and tag prefixes defined, however these columns are not present in my data source file. When I run the Ruby script the non-matched tags are not added to the JSON, while the Python script includes the tags and tag prefixes from the metadata file, even though the data source file does not have column matches for the tag columns provided in the metadata file.
[testing.zip](https://github.com/KennaSecurity/All_Samples/files/9985285/testing.zip)

File: KDI Importer/README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+[Kenna Data Importer (KDI)](https://help.kennasecurity.com/hc/en-us/articles/360026413111-Kenna-Data-Importer-JSON-Connector-).. 
+This copy has been updated to optionally import asset data only.
 
-**UPDATE: This has been updated from our original KDI Transformer script that has been updated to help with Asset-Only imports.**
-
-Note: This script can still be used like the original, but just omitting the last two parameters when running.
-
-##  Usage
+## Ruby Usage
 ```
```
Comment (victorz1982) [inline KDI Importer/README.md:8]:
```suggestion
```ruby
```
See [Syntax highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) in markdown docs

File: KDI Importer/README.md [hunk 2]
```diff
@@ -18,12 +16,45 @@ csv_KDI_json.rb source_vuln_file.csv has_header? metafile.csv skip_autoclose? ou
 - assets_only? => OPTIONAL parameter to indicate to translate ONLY ASSET DATA. Either "true" or "false"; defaults to false
 - domain_suffix? => OPTIONAL parameter to provide a domain suffix to append to hostnames. (Still experimental)
 
+## Python Usage
+```
```
Comment (victorz1982) [inline KDI Importer/README.md:20]:
```suggestion
```python
```
Similar to the other suggestion

**Review by victorz1982 — APPROVED** [review]
There are 2 non-blocking comments

**Review by jaredkalmus — APPROVED** [review]

**Pull request merged**